### DISCUSSION TITLE
MPD Page not available

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#33] [Fixed Issues]
Background: I am trying to use my PI Zero W, which has no sound card or DAC attached,  to output directly via Bluetooth to my SoundStick BT speakers. I can connect to the speakers with variable results as reported elsewhere in other Discussion items (#12)

After applying update 20201228 the MPD page is unavailable. If I temporarily assign any I2S Audio device via the System page  (despite none actually being attached to the RPi Zero) then the MPD page can be accessed but this page has the same problems reported in #26.

Chris

**Comment by rern** [comment]
Fixed.
